spec/requests/v1/pacbio/runs_spec.rb

```diff
@@ -83,6 +82,16 @@
       expect(json['data'][0]['relationships']['smrt_link_version']).to be_present
       expect(json['data'][0]['relationships']['smrt_link_version']['data']['type']).to eq 'smrt_link_versions'
       expect(json['data'][0]['relationships']['smrt_link_version']['data']['id']).to eq run1.smrt_link_version.id.to_s
+
+      annotation_attributes = find_included_resource(type: 'annotations', id: run1.annotations.first.id)['attributes']
+      expect(annotation_attributes).to include(
+        'comment' => run1.annotations.first.comment,
+        'user' => run1.annotations.first.user,
+        'created_at' => run1.annotations.first.created_at.to_fs(:us),
+        'annotation_type_id' => run1.annotations.first.annotation_type_id,
+        'annotatable_type' => run1.annotations.first.annotatable_type,
+        'annotatable_id' => run1.annotations.first.annotatable_id
+      )
```

Comment:
Do we need to have a test cases to test the deletion of run deletes the associated annotations and update of run with updated annotations?

Reply:
Also, for the wells added, removed and updated with annotations?